superset-frontend/src/SqlLab/reducers/getInitialState.ts

```diff
@@ -140,11 +136,12 @@ export default function getInitialState({
    * hasn't used SQL Lab after it has been turned on, the state will be stored
    * in the browser's local storage.
    */
-  if (
-    localStorage.getItem('redux') &&
-    JSON.parse(localStorage.getItem('redux')).sqlLab
-  ) {
-    const { sqlLab } = JSON.parse(localStorage.getItem('redux'));
+  const localStorageData = localStorage.getItem('redux');
```

Comment:
Once we consolidate the redux store with SPA store this (redux) will cover for all redux persist state but I agree with the naming. The problem is we should keep checking `redux` value because the legacy user still stores the previous session data in that key. so it's not hard but hard to maintain/manage new key value and old key value.